src/main/resources/youtube/settings/host/values/strings.xml

```diff
@@ -378,26 +378,26 @@ Some components may not be hidden."</string>
     <string name="revanced_hide_navigation_home_button_summary_off">Home button is shown.</string>
     <string name="revanced_hide_navigation_home_button_summary_on">Home button is hidden.</string>
-    <string name="revanced_hide_navigation_library_button_title">Hide library button</string>
+    <string name="revanced_hide_navigation_library_button_title">Hide Library button</string>
     <string name="revanced_hide_navigation_library_button_summary_on">Library button is hidden.</string>
     <string name="revanced_hide_navigation_library_button_summary_off">Library button is shown.</string>
     <string name="revanced_hide_navigation_notifications_button_title">Hide notifications button</string>
     <string name="revanced_hide_navigation_notifications_button_summary_on">Notifications button is hidden.</string>
     <string name="revanced_hide_navigation_notifications_button_summary_off">Notifications button is shown.</string>
-    <string name="revanced_hide_navigation_shorts_button_title">Hide shorts button</string>
+    <string name="revanced_hide_navigation_shorts_button_title">Hide Shorts button</string>
     <string name="revanced_hide_navigation_shorts_button_summary_on">Shorts button is hidden.</string>
     <string name="revanced_hide_navigation_shorts_button_summary_off">Shorts button is shown.</string>
-    <string name="revanced_hide_navigation_subscriptions_button_title">Hide subscriptions button</string>
+    <string name="revanced_hide_navigation_subscriptions_button_title">Hide Subscriptions button</string>
     <string name="revanced_hide_navigation_subscriptions_button_summary_on">Subscriptions button is hidden.</string>
     <string name="revanced_hide_navigation_subscriptions_button_summary_off">Subscriptions button is shown.</string>
-    <string name="revanced_hide_navigation_label_title">Hide navigation label</string>
-    <string name="revanced_hide_navigation_label_summary_on">Navigation label is hidden.</string>
-    <string name="revanced_hide_navigation_label_summary_off">Navigation label is shown.</string>
-    <string name="revanced_switch_create_with_notifications_button_title">Switch create with notifications</string>
-    <string name="revanced_switch_create_with_notifications_button_summary">"Swaps the positions of the create button and notification button by spoofing device information.
+    <string name="revanced_hide_navigation_label_title">Hide navigation labels</string>
+    <string name="revanced_hide_navigation_label_summary_on">Navigation labels are hidden.</string>
+    <string name="revanced_hide_navigation_label_summary_off">Navigation labels are shown.</string>
+    <string name="revanced_switch_create_with_notifications_button_title">Swap create and Notifications buttons</string>
+    <string name="revanced_switch_create_with_notifications_button_summary">"Swaps the positions of the create button with the Notifications button by spoofing device information.
```

Comment:
Here too I did not capitalize "create"

Reply:
In ReVanced:
https://github.com/ReVanced/revanced-patches/blob/b72fe87cf9b0a5d8105ead9b50f0c72cf8cc4c97/src/main/resources/addresources/values/strings.xml#L437